src/utils/dateTimeUtil.ts

```diff
@@ -0,0 +1,16 @@
+import moment from 'moment-timezone';
+import _ from 'lodash';
+
+//accepts a date-time string and outputs a human readable string with timezone
+//output format 31/12/1990 11:59 pm IST
+const getDateTimeWithTZ = (dateTime: string | undefined) => {
+	if (!dateTime && typeof dateTime !== 'string' && _.isEmpty(dateTime)) return;
```

Comment:
refactor and simplify this condition. not readable (use combination of || and &&)